_posts/2020-12-24-JabRef5-2.md

```diff
@@ -0,0 +1,35 @@
+---
+title:  🎄 JabRef 5.2 Release 🎄
+id: jabref-5-2
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)"
+---
+
+We are proud to announce the release of the next milestone of JabRef. Today we are releasing JabRef 5.2. Just in time for the holidays!
+We are also happy to announce that we have a new developer, [Domninik Voigt](https://github.com/DominikVoigt), who implemented a proof of concept tool for systematic literature search in JabRef as part of his thesis. The feature still needs some work until it's usable for non-technical users.
+
+## Notable Changes and New Features
+
+After the release of the previous major milestone 5.1 we focused on bug fixes and some new features!
+
+This release brings some improvements and bug fixes for users who are using shared SQL databases for collaborative work and those that use LibreOffice.
+For users switching from EndNote to JabRef, we improved the importer. We fixed a bug where possible errors during the import were not shown to the user. In addition, we now added a YAML exporter to export your library in the YAML format.
+
+Additionally, to these improvements under the hood, we also tweaked the user interface. In 5.0 we introduced the option to have a “Dark Theme” and with this release it is now possible to create your own theme by [adjusting the CSS](https://docs.jabref.org/advanced/custom-themes).
+The preferences dialog and some menu items were adjusted for consistency. For those that missed our Emacs/bash keybindings, we are happy to tell you that we reimplemented this feature. The feature can now be configured as part of the "Manage keybindings" dialog.
+
+The JabRef Browser extension is now also compatible with the new version of Microsoft Edge and avaiable on all
+
+[And many more! In total, there are again many changes](https://github.com/JabRef/jabref/blob/master/CHANGELOG.md) since our last official release!
+
+**You can get it from [FOSShub](https://www.fosshub.com/JabRef.html).**
+
+Please report any other issues that you encounter in our [bug tracker](https://github.com/JabRef/jabref/issues) (in case it was not already reported before).
+
+## Special Thanks
+
+The [JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS) would like to give special thanks to all new and old contributors. We are very happy to see so many new contributors opening pull requests and making translations. As we all work on JabRef in our free time, we feel honored that people share our enthusiasm for free software and work together to make research easier. Some big news about how to be part of this project will follow soon.
+
+We also want to thank all universities [including JabRef in their academic teaching](https://devdocs.jabref.org/teaching).
```

Comment:
Should we mention Basel directly?

Reply:
Let's ask @koppor

Reply:
There are no contributions in `master` - will be included in `5.3` (hopefully)